datalad_container/containers_list.py

```diff
@@ -75,6 +75,8 @@ def __call__(dataset=None, recursive=False):
             if 'image' not in v:
                 # there is no container location configured
                 continue
+            if dataset and 'dspath' not in v:
+                v['dspath'] = dataset.path
```

Comment:
I think this would be better as

```diff
diff --git a/datalad_container/containers_list.py b/datalad_container/containers_list.py
index cda137c..b08eff0 100644
--- a/datalad_container/containers_list.py
+++ b/datalad_container/containers_list.py
@@ -75,13 +75,12 @@ def __call__(dataset=None, recursive=False):
             if 'image' not in v:
                 # there is no container location configured
                 continue
-            if dataset and 'dspath' not in v:
-                v['dspath'] = dataset.path
             res = get_status_dict(
                 status='ok',
                 action='containers',
                 name=k,
                 type='file',
+                parentds=ds.path,
                 path=op.join(ds.path, v.pop('image')),
                 # TODO
                 #state='absent' if ... else 'present'
```

Changes:

  * Drop condition on `dataset` and instead use resolved `ds`.  Otherwise the value won't be set for images in the top-level dataset if an explicit dataset isn't supplied.
  * Don't bother checking for the value in config dict because (1) we don't save this value and (2) I can't see why the user should be able to manually override this value in the config.
  * Call the field `parentds` for consistency with the field in `status`.

Thoughts?

Reply:
Thanks for the review/suggestions!  Re `parentds` - that seems would make it also consistent with `metadata`'s records.  
I wondered if then `img_dspath` should be longer/uglier `img_parentdspath` for having clear association. But since we are abbreviating anyways, I guess we will just consider `dspath` to be ok there.
Anyways -- will need to adjust `containers-add --help` to mention this new placeholder.